tests/conditionalTest.js

```diff
@@ -0,0 +1,5 @@
+const conditionalTest = (title, testLambda, condition) => {
```

Comment:
this is nice... I would try to think of a better way, so we won't have to remember adding 'conditionalTest' instead of just 'test' everytime we add a new test. But this is not critical for now, we / I can refactor later...